webapp/client/public/css/base.css

```diff
@@ -30,7 +34,7 @@ html,
 body {
   margin: 0 !important;
   padding: 0 !important;
-  font-family: "BundesSansWeb", sans-serif;
+  font-family: var(--font-main);
```

Comment:
I  added some fallbacks in case the BundesSansWeb font could not be loaded. It is recommended to do that, but maybe the UX colleagues want different fallback fonts, which we can change later.

Reply:
Yes, I already asked UX to have a look and we will talk about